packages/backend/src/execution/executors/PgExecutor/PgExecutor.ts

```diff
@@ -8,6 +8,19 @@ import { composeQuery } from './composeQuery';
 import { hydrate } from './hydrate';
 import { SynthqlError } from '../../../SynthqlError';
 
+const TIME_OID = types.builtins.TIME;
+const TIMETZ_OID = types.builtins.TIMETZ;
+const DATE_OID = types.builtins.DATE;
+const TIMESTAMP_OID = types.builtins.TIMESTAMP;
+const TIMESTAMPTZ_OID = types.builtins.TIMESTAMPTZ;
```

Comment:
https://discord.com/channels/1222945509266817055/1222945509740642317/1273354417638342687

Reply:
No useless indirections please. Read your code before you ask for a review. 

Please make a conscious effort to build that habit.